sqlmesh/integrations/github/cicd/options.py

```diff
@@ -0,0 +1,19 @@
+import click
+
+token = click.option(
```

Comment:
i don't think we need this, we're not gonna have that many commands / it being reusable, makes it harder to manage, move it into the cli file